ibu-imager/installation_configuration_files/scripts/installation-configuration.sh

```diff
@@ -169,27 +196,11 @@ until openssl x509 -in ${KUBELET_CLIENT_CERTIFICATE} -checkend 30 &> /dev/null;
 done
 echo "${KUBELET_CLIENT_CERTIFICATE} is valid."
 
-# Reconfigure DNS
-node_ip=$(oc get nodes -o jsonpath='{.items[0].status.addresses[?(@.type == "InternalIP")].address}')
-domain="$(oc apply -f "${RELOCATION_CONFIG_PATH}" --dry-run=client -o jsonpath='{.items[?(@.kind=="ClusterRelocation")].spec.domain}')"
-
-if [ -z ${domain+x} ]; then
-    echo "domain not defined"
-else
-    echo "Updating dnsmasq with new domain"
-    cat << EOF > /etc/dnsmasq.d/customer-domain.conf
-address=/apps.${domain}/${node_ip}
-address=/api-int.${domain}/${node_ip}
-address=/api.${domain}/${node_ip}
-EOF
-    systemctl restart dnsmasq
-fi
-
 echo "Applying cluster configuration"
 oc apply -f "${RELOCATION_CONFIG_PATH}"
-echo "Waiting for cluster relocation status"
-oc wait --timeout=1h clusterrelocation cluster --for condition=Reconciled=true &> /dev/null
-echo "Cluster configuration updated"
+#echo "Waiting for cluster relocation status"
+#oc wait --timeout=1h clusterrelocation cluster --for condition=Reconciled=true &> /dev/null
+#echo "Cluster configuration updated"
 
```

Comment:
We wipe /opt/openshift at the end, after successful completion. Should we also delete the env file (and other injected files) as part of cleanup? Or this could be done as part of the post-reboot completion of the Upgrade stage in LCA.
```
/etc/systemd/system/installation-configuration.env
/etc/systemd/system/installation-configuration.service
/etc/systemd/system/prepare-installation-configuration.service
/usr/local/bin/installation-configuration.sh
/usr/local/bin/prepare-installation-configuration.sh
```

Reply:
We can do it as a next step

Reply:
Will open task for cleanup